en/SUMMARY.md

```diff
@@ -9,7 +9,7 @@
   * [Scripts/Examples](getting_started/examples_scripts.md)
   * [Generate Source Files for ROS](getting_started/ros.md)
 * [MAVLink Versions](guide/mavlink_version.md)
-  * [MAVLink 2](guide/mavlink_2.md)
+* [MAVLink 2](guide/mavlink_2.md)
```

Comment:
Note, now at same level. Propose to move this and versions topic into Guide, along with Serialisation etc. Will keep message definitions out of that guide.